### ISSUE TITLE
When using sdk touch event not fire

### ISSUE THREAD

**Issue by TMily** [microsoft/appcenter-sdk-dotnet#1289]
### **Description**

When i using sdk in my app touch event will be not fire

### **Repro Steps**

Please list the steps used to reproduce your issue.

1.Init sdk in App_OnStartup
2.touch screen 

### **Details**

1. What is your app platform (Xamarin.Android or Xamarin.iOS or UWP)?
   - .Net Core 3.0 WPF Application
2. Which SDK version are you using?
    - 2.6.4
4. Which OS version did you experience the issue on?
    - Windows 10 1909
5. What device version did you see this error on?  Were you using an emulator or a physical device?
    - PC
6. What third party libraries are you using? 
    - Include="Aliyun.Api.LogService" Version="1.1.0" 
    - Include="Autofac" Version="4.9.4" 
    - Include="Automatonymous" Version="4.2.0" 
    - Include="EFCore.BulkExtensions" Version="3.1.0" 
    - Include="Grpc.Net.Client" Version="2.26.0" 
    - Include="MaterialDesignColors" Version="1.2.2" 
    - Include="MaterialDesignThemes" Version="3.0.1" 
    - Include="Microsoft.AppCenter.Analytics" Version="2.6.4" 
    - Include="Microsoft.AppCenter.Crashes" Version="2.6.4" 
    - Include="Microsoft.AppCenter.Distribute" Version="2.6.4" 
    - Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="2.9.8"
    - Include="Microsoft.EntityFrameworkCore.Sqlite" Version="3.1.0" 
    - Include="Microsoft.Extensions.Configuration" Version="3.1.0" 
    - Include="Microsoft.Extensions.Configuration.Binder" Version="3.1.0" 
    - Include="Microsoft.Extensions.Configuration.EnvironmentVariables" Version="3.1.0" 
    - Include="Microsoft.Extensions.Configuration.Json" Version="3.1.0" 
    - Include="NLog" Version="4.6.8" 
    - Include="RestSharp" Version="106.10.1" 
    - Include="RocksDbNative" Version="6.2.2" 
    - Include="RocksDbSharp" Version="6.2.2" 
    - Include="sqlite-net-pcl" Version="1.7.302-beta" 
    - Include="SQLitePCLRaw.lib.e_sqlite3.v110_xp" Version="1.1.14" 
    - Include="System.IO.Pipelines" Version="4.7.0" 
    - Include="System.IO.Ports" Version="4.7.0" 
    - Include="System.Runtime" Version="4.3.1" 
    - Include="System.Text.Encoding.CodePages" Version="4.7.0" 
    - Include="System.Text.Json" Version="4.7.0" 
7. Please enable verbose logging for your app using `AppCenter.LogLevel = LogLevel.Verbose` before your call to `AppCenter.Start(...)` and include the logs here:
<details>
  <summary> Logs </summary>

```
2020-01-14 18:04:33.525 [AppCenter] DEBUG: No named identifier found in appSecret; using as-is
2020-01-14 18:04:33.589 [AppCenter] DEBUG: Enabled TLS 1.2 explicitly as it was disabled.
2020-01-14 18:04:33.672 [AppCenter] DEBUG: AddChannel(core)
2020-01-14 18:04:33.875 [AppCenter] INFO: App Center SDK configured successfully.
2020-01-14 18:04:33.880 [AppCenter] DEBUG: AddChannel(analytics)
2020-01-14 18:04:33.884 [AppCenterAnalytics] DEBUG: SessionTracker.Resume
2020-01-14 18:04:33.890 [AppCenter] DEBUG: Invoke EnqueuingLog event for channel 'analytics'
2020-01-14 18:04:34.306 [AppCenter] DEBUG: Invoke FilteringLog event for channel 'analytics'
2020-01-14 18:04:34.323 [AppCenter] INFO: 'Analytics' service started.
2020-01-14 18:04:34.329 [AppCenter] DEBUG: AddChannel(crashes)
2020-01-14 18:04:34.542 [AppCenter] DEBUG: CheckPendingLogs(analytics) pending log count: 1
2020-01-14 18:04:34.547 [AppCenter] INFO: 'Crashes' service started.
2020-01-14 18:04:34.549 [AppCenter] DEBUG: Invoke EnqueuingLog event for channel 'core'
2020-01-14 18:04:34.805 [AppCenter] DEBUG: Invoke FilteringLog event for channel 'core'
2020-01-14 18:04:34.826 [AppCenter] DEBUG: CheckPendingLogs(core) pending log count: 1
2020-01-14 18:04:40.123 [AppCenter] DEBUG: TriggerIngestion(core) pending log count: 1
2020-01-14 18:04:40.126 [AppCenter] DEBUG: TriggerIngestion(analytics) pending log count: 1
2020-01-14 18:04:40.154 [AppCenter] DEBUG: Trying to get up to 50 logs from storage for core
2020-01-14 18:04:40.249 [AppCenter] DEBUG: The SID/ID pairs for returning logs are:
	(null) / 8
2020-01-14 18:04:40.251 [AppCenter] DEBUG: Trying to get up to 50 logs from storage for analytics
2020-01-14 18:04:40.254 [AppCenter] DEBUG: Invoke SendingLog event for channel 'core'
2020-01-14 18:04:40.272 [AppCenter] DEBUG: The SID/ID pairs for returning logs are:
	f495b50e-fa67-4250-8439-a44482d0084b / 7
2020-01-14 18:04:40.274 [AppCenter] DEBUG: Invoke SendingLog event for channel 'analytics'
2020-01-14 18:04:42.308 [AppCenter] VERBOSE: Calling https://in.appcenter.ms/logs?api-version=1.0.0...
2020-01-14 18:04:42.311 [AppCenter] VERBOSE: Headers: App-Secret=****************************e353aa70, Install-ID=4c3fc2c1-20e6-474d-9ce1-bd0c7831d5e1
2020-01-14 18:04:42.316 [AppCenter] VERBOSE: {
  "logs": [
    {
  "services": [
    "Analytics",
    "Crashes"
  ],
  "timestamp": "2020-01-14T10:04:34.8055313Z",
  "device": {
    "sdkName": "appcenter.wpf.netcore",
    "sdkVersion": "2.6.4",
    "model": "To be filled by O.E.M.",
    "oemName": "Gigabyte Technology Co., Ltd.",
    "osName": "WINDOWS",
    "osVersion": "10.0.19041",
    "osBuild": "10.0.19041.1",
    "locale": "zh-CN",
    "timeZoneOffset": 480,
    "screenSize": "2560x1440",
    "appVersion": "1.0.0",
    "appBuild": "1.0.0",
    "appNamespace": "EPosCore"
  },
  "type": "startService"
}
  ]
}
2020-01-14 18:04:42.370 [AppCenter] VERBOSE: Calling https://in.appcenter.ms/logs?api-version=1.0.0...
2020-01-14 18:04:42.373 [AppCenter] VERBOSE: Headers: App-Secret=****************************e353aa70, Install-ID=4c3fc2c1-20e6-474d-9ce1-bd0c7831d5e1
2020-01-14 18:04:42.376 [AppCenter] VERBOSE: {
  "logs": [
    {
  "timestamp": "2020-01-14T10:04:34.3063179Z",
  "sid": "f495b50e-fa67-4250-8439-a44482d0084b",
  "device": {
    "sdkName": "appcenter.wpf.netcore",
    "sdkVersion": "2.6.4",
    "model": "To be filled by O.E.M.",
    "oemName": "Gigabyte Technology Co., Ltd.",
    "osName": "WINDOWS",
    "osVersion": "10.0.19041",
    "osBuild": "10.0.19041.1",
    "locale": "zh-CN",
    "timeZoneOffset": 480,
    "screenSize": "2560x1440",
    "appVersion": "1.0.0",
    "appBuild": "1.0.0",
    "appNamespace": "EPosCore"
  },
  "type": "startSession"
}
  ]
}
2020-01-14 18:04:42.453 [AppCenter] DEBUG: CheckPendingLogs(analytics) pending log count: 0
2020-01-14 18:04:42.455 [AppCenter] DEBUG: CheckPendingLogs(core) pending log count: 0
2020-01-14 18:04:44.362 [AppCenter] VERBOSE: HTTP response status=200 (OK) payload={"status":"Success","validDiagnosticsIds":[],"throttledDiagnosticsIds":[],"correlationId":"de9c4c29-57b3-4ada-93c5-75c40d3986c1"}
2020-01-14 18:04:44.366 [AppCenter] VERBOSE: HTTP response status=200 (OK) payload={"status":"AllLogsThrottled","validDiagnosticsIds":[],"throttledDiagnosticsIds":[],"correlationId":"df7e3d65-8a22-4514-9a9d-0ab33ff1aee8"}
2020-01-14 18:04:44.374 [AppCenter] DEBUG: Invoke SentLog event for channel 'analytics'
2020-01-14 18:04:44.376 [AppCenter] DEBUG: Invoke SentLog event for channel 'core'
2020-01-14 18:04:44.385 [AppCenter] DEBUG: CheckPendingLogs(core) pending log count: 0
2020-01-14 18:04:44.389 [AppCenter] DEBUG: CheckPendingLogs(analytics) pending log count: 0
2020-01-14 18:04:44.396 [AppCenter] DEBUG: Deleting logs from storage for channel 'analytics' with batch id '28b30659-2ec1-41e1-aaec-a048daf4a933'
2020-01-14 18:04:44.400 [AppCenter] DEBUG: The IDs for deleting log(s) is/ are:
	7
2020-01-14 18:04:44.419 [AppCenter] DEBUG: Deleting logs from storage for channel 'core' with batch id '250d9786-c4a0-4137-8fd6-b041d82a2b39'
2020-01-14 18:04:44.422 [AppCenter] DEBUG: The IDs for deleting log(s) is/ are:
	8
```

</details>

**Comment by ilyashumihin** [comment]
Hi @TMily!
Thanks for getting in touch!

We were able to reproduce this bug, thanks for the report.

**Comment by guperrot** [comment]
This issue has been fixed in version [3.0.0](https://github.com/microsoft/appcenter-sdk-dotnet/releases/3.0.0).
